assets/sass/modules/_qtip.scss

```diff
@@ -36,3 +36,50 @@
     }
   }
 }
+
+/*! Custom qtip styles, based on dark theme */
+.qtip-custom {
```

Comment:
wonder the scss lint shall generate an warning that combine line 7 and line 41-85 since they all `.qtip-custom` ?